nosql/assignment.nosql.schema.txt

```diff
@@ -0,0 +1,84 @@
+
+Collection - Users
+
+{
+	account: {
+		username: "shirish.sp",
```

Comment:
It;s fine to have username as a primary/unique key, but for reference its always better to use the collection's _id, since it is indexed. Faster retrieval.